### ISSUE TITLE
PBS walltime limits

### ISSUE THREAD

**Issue by arahuja** [hammerlab/stratocumulus#52]
Is this set somewhere or a default?

I hit `=>> PBS: job killed: walltime 86430 exceeded limit 86400`, making this configurable or larger would be nice, this is currently 24hr

**Comment by arahuja** [comment]
I see this actually a ketrew issue, and possibly already configurable:
https://github.com/hammerlab/ketrew/pull/443/files

**Comment by smondet** [comment]
Yes if you're using the `stratotemplate` you just have to modify the `biokepi_machine.ml`.
